core/file_api.php

```diff
@@ -1081,7 +1081,7 @@ function file_add( $p_bug_id, array $p_file, $p_table = 'bug', $p_title = '', $p
 
 	$t_file_info['id'] = db_insert_id( $t_file_table );
 
-	if( db_is_oracle() ) {
+	if( db_is_oracle() || db_is_sqlite3() ) {
```

Comment:
Note that *db_update_blob()* is also called from mc_file_api.php.

Maybe we should introduce a new, more functionally-oriented check, like db_uses_blob() returning true for oci8 and sqlite, instead of checking both db_is_oracle() / db_is_sqlite3() every time.

Reply:
> Note that db_update_blob() is also called from mc_file_api.php.

Added.

Reply:
I liked the clever approach with db_uses_blob(), added it.